### DISCUSSION TITLE
Confused about duplicate arguments?

### DISCUSSION THREAD

**Discussion by secretmango** [fwupd/fwupd#5624] [General]
**Describe the question**
There are lots of arguments available, while the howto states only a few as nessecary. What are duplicates?

For example upgrade, install

refresh, update, get-updates, get-upgrades

what do they do?

```
activate              esp-mount             firmware-read         get-report-metadata   --no-safety-check     smbios-dump
--allow-older         esp-unmount           firmware-sign         get-topology          --no-search           switch-branch
--allow-reinstall     export-hwids          --force               get-updates           --plugins             unbind-driver
attach                --filter              get-bios-setting      get-upgrades          --prepare             update
bind-driver           firmware-build        get-details           hwids                 refresh               upgrade
build-firmware        firmware-convert      get-device-flags      --ignore-checksum     reinstall             --verbose
--cleanup             firmware-dump         get-devices           --ignore-vid-pid      --save-backends       verify-update
clear-history         firmware-export       get-firmware-types    install               security              watch
detach                firmware-extract      get-history           install-blob          self-sign             
--disable-ssl-strict  firmware-parse        get-plugins           --method              set-bios-setting      
esp-list              firmware-patch        get-remotes           monitor               --show-all 
```

**Comment by hughsie** [comment]
The fwupdtool is a hardcore developer and debugging tool, and not designed for end-users. Users should use fwupdmgr which has a lot less options. That said, the man page does suck and we're working on making that better.

**Comment by secretmango** [comment]
ok didnt know that! So I will use fwupdmgr (which btw is pretty hard to spell). Maybe changing the name to something more spellable? fwupd is already kinda hard ;D

great tool, thanks a lot for your work!

**Comment by superm1** [comment]
> The fwupdtool is a hardcore developer and debugging tool, and not designed for end-users. Users should use fwupdmgr which has a lot less options. That said, the man page does suck and we're working on making that better.

The above doesn't look like a man page or even --help output. It's bash completion.
